docs/guide/execute-transaction.md

```diff
@@ -143,7 +142,33 @@ Examples of parameter [`ExecParams`](https://algobuilder.dev/api/algob/modules/r
     sign: SignType.SecretKey,
     fromAccount: john,
     appId: 10,
-    payFlags: { totalFee: 1000 },
+    payFlags: { totalFee: fee },
     appArgs: []
   }
 ```
+
+### Pooled Transaction Fees
+
+With [this](https://developer.algorand.org/articles/introducing-algorand-virtual-machine-avm-09-release/) release, algob also supports pooled transaction fees.
+Supports pooled fees where one transaction can pay the fees of other transactions within an atomic group. For atomic transactions, the protocol sums the number of transactions and calculates the total amount of required fees, then calculates the amount of fees submitted by all transactions. If the collected fees are greater than or equal to the required amount, the transaction fee requirement will be met.
+Ex:
+```js
+  {
+    type: TransactionType.TransferAlgo,
+    sign: SignType.SecretKey,
+    fromAccountAddr: john,
+    toAccountAddr: alice.address,
+    amountMicroAlgos: 100,
+    payFlags: { totalFee: 2000 }
+  },
+  {
+    type: TransactionType.TransferAlgo,
+    sign: SignType.SecretKey,
+    fromAccountAddr: alice,
+    toAccountAddr: bob.address,
+    amountMicroAlgos: 100,
+    payFlags: { totalFee: 0 }
+  }
+```
+
+Even though fee paid by alice is `0`, this transaction will pass because total fees collected is greater then minimum required
```

Comment:
> total fees collected is greater then minimum required

should this be greator than or equal to ? Or is it strictly greator than?

Reply:
`greater than or equal to`